Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/StatisticsLayoutManager.kt

```diff
@@ -0,0 +1,43 @@
+package de.rki.coronawarnapp.statistics.ui.homecards
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+
+class StatisticsLayoutManager : LinearLayoutManager {
```

Comment:
Could you please add a comment above why this is needed.

Reply:
yes